watson_developer_cloud/speech_to_text_v1.py

```diff
@@ -18,10 +18,133 @@
 
 from .watson_developer_cloud_service import WatsonDeveloperCloudService
 import json
+from collections import deque, Counter, namedtuple
+import base64
+
+from autobahn.twisted.websocket import WebSocketClientFactory,\
+    WebSocketClientProtocol, connectWS
+from twisted.internet import ssl, reactor, task, interfaces
+from zope.interface import implementer
+
+Filedata = namedtuple('FileData', ['file_name',
+                                   'file_metadata',
+                                   'initial_data'])
+
+
+@implementer(interfaces.IPushProducer)
+class AudioFileProducer:
+    def __init__(self, proto):
+        self.proto = proto
+        self.started = False
+        self.paused = False
+        self.filedata = proto.filedatasource.pop()
+        self.fileobj = open(self.filedata.file_name, 'rb')
+        self.proto.metadata = self.filedata.file_metadata
+
+    def pauseProducing(self):
```

Comment:
should be `snake_case` and probably just `pause` since you will call the variable `producer` you can say

`producer.pause()`